meta-sparrow/recipes-nemomobile/usb-moded/usb-moded/init_gfs

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+serial="$(cat /proc/cmdline | sed 's/.*androidboot.serialno=//' | sed 's/ .*//')"
+
+mkdir -p /config
+mount -t configfs none /config
+mkdir -p /config/usb_gadget/g1
+
+echo 0x0e8d > /config/usb_gadget/g1/idVendor
+echo 0x0010 > /config/usb_gadget/g1/idProduct
+
+mkdir /config/usb_gadget/g1/strings/0x409
+echo $serial > /config/usb_gadget/g1/strings/0x409/serialnumber
+echo AsteroidOS > /config/usb_gadget/g1/strings/0x409/manufacturer
+hostname > /config/usb_gadget/g1/strings/0x409/product
+
+mkdir -p /config/usb_gadget/g1/configs/b.1/strings/0x409
+mkdir -p /config/usb_gadget/g1/functions/
+
+mkdir /config/usb_gadget/g1/functions/rndis.usb0
+mkdir /config/usb_gadget/g1/configs/c.1
+mkdir /config/usb_gadget/g1/configs/c.1/strings/0x409
+echo "rndis" > /config/usb_gadget/g1/configs/c.1/strings/0x409/configuration
+
+ln -s /config/usb_gadget/g1/functions/rndis.usb0 /config/usb_gadget/g1/configs/c.1
+echo "$(ls /sys/class/udc)" > /config/usb_gadget/g1/UDC
+
+ifconfig usb0 192.168.2.15
+
+while true; do
```

Comment:
This stuff isn't great, but `usb-moded` can be a pain to get working properly as we've seen before.

Reply:
I added a comment to make it more explicit which part should really be done by usb-moded ultimately and that this is terribly wrong and fixing it is left as an exercise to the reader.